lib/l10n/app_zh.arb

```diff
@@ -0,0 +1,185 @@
+{
+    "@@locale": "zh",
+
+    "continueWatching": "继续观看",
+    "recentlyAddedMovies": "最近添加的电影",
+    "recentlyAddedShows": "最近添加的剧集",
+    "yourWatchlist": "你的观看列表",
+    "top10inYourLibrary": "您库里的前十榜单",
+    "similarToWatchHistory": "与您最近看过的相似",
+    "becauseYouWatched": "因为您看过 {title}",
+    "@becausYouWatched": {
+        "placeholders": {
+          "title": {
+            "type": "String",
+            "example": "Forrest Gump"
+          }
+        }
+      },
+    "becauseYouLiked": "因为您喜欢 {title}",
+    "@becauseYouLiked": {
+        "placeholders": {
+          "title": {
+            "type": "String",
+            "example": "Forrest Gump"
+          }
+        }
+      },
+    "directedBy": "由 {director} 导演",
+    "@directedBy": {
+        "placeholders": {
+          "director": {
+            "type": "String",
+            "example": "Robert Zemeckis"
+          }
+        }
+      },
+    "starring": "由 {actor} 参演",
+    "@starring": {
+        "placeholders": {
+          "actor": {
+            "type": "String",
+            "example": "Tom Hanks"
+          }
+        }
+      },
+    "recommendedForYou": "为您推荐",
+    "play": "播放",
+    "moreInfo": "详情",
+    "search": "搜索",
+    "startTypingSearch": "开始输入以搜索",
+    "genre": "类型",
+    "all": "所有",
+    "filter": "过滤",
+    "none": "无",
+    "order": "顺序",
+    "ascending": "升序",
+    "descending": "降序",
+    "sort": "排序",
+    "name": "名称",
+    "premiereDate": "首映日期",
+    "random": "随机",
+    "unplayed": "没看过",
+    "played": "看过",
+    "favorites": "最爱的",
+    "liked": "喜欢的",
+    "changeProfile": "切换用户",
+    "logout": "登出",
+    "about": "关于",
+    "scanLibrary": "扫描库",
+    "scanStarted": "扫描已开始",
+    "quickConnect": "快速连接",
+    "quickConnectDescription": "输入你设备上显示的代码",
+    "quickConnectError401": "你访问的是正确的服务器吗？",
+    "quickConnectError404": "无效代码",
+    "quickConnectErrorUnknown": "Something went wrong",

```

Comment:
Could you please remove the untranslated messages or translate them. The app will use the fallback language which is english now.

Reply:
I amended the PR, it should now also contain the translation of this line (and also other previously untranslated lines).